README.md

```diff
@@ -1 +1,19 @@
-# gen-ai-examples
\ No newline at end of file
+# Gen-AI Examples
+
+This repository contains a collection of examples demonstrating the usage of the GDP Labs GenAI SDK. These examples are intended to help developers understand the SDK’s capabilities and learn how to integrate them into their own projects.
+
+> [!WARNING]
+> This SDK is currently in binary version and is only available for Linux.
```

Comment:
Let's change this to `This SDK is in binary version and is currently only available for Linux.`

Because we don't plan to publish the plain source-code (non-binary) version.

Reply:
thanks for the wording improvement suggestion, updated here [5448566](https://github.com/GDP-ADMIN/gen-ai-examples/pull/2/commits/5448566129942152ea5ee9258ff69d127b4f23af)